airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+import math
+from dataclasses import dataclass
+from typing import Any, Iterable, List, Mapping, Optional
+
+import requests
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.requesters.paginators.strategies.page_increment import PageIncrement
+from airbyte_cdk.sources.declarative.stream_slicers.datetime_stream_slicer import DatetimeStreamSlicer
+from airbyte_cdk.sources.declarative.stream_slicers.substream_slicer import SubstreamSlicer
+from airbyte_cdk.sources.declarative.types import StreamSlice, StreamState
+
+
+@dataclass
+class ZenloopDatetimeStreamSlicer(DatetimeStreamSlicer):
```

Comment:
do we need this class or could we use the default `DatetimeStreamSlicer` and add a request param in the requester's config?

```
request_options_provider:
  request_parameters:
    order_type: "desc"
    order_by: "inserted_at"
    "date_shortcut": "custom"
```

Reply:
Yes, we need this class. The reason is that we need ` "date_shortcut": "custom"` param only during incremental sync. More context above - https://github.com/airbytehq/airbyte/pull/19624#discussion_r1029881973

Reply:
Updated.